doc/source/whatsnew/v0.24.0.rst

```diff
@@ -914,6 +914,7 @@ update the ``ExtensionDtype._metadata`` tuple to match the signature of your
 - Added ``ExtensionDtype._is_numeric`` for controlling whether an extension dtype is considered numeric (:issue:`22290`).
 - The ``ExtensionArray`` constructor, ``_from_sequence`` now take the keyword arg ``copy=False`` (:issue:`21185`)
 - Bug in :meth:`Series.get` for ``Series`` using ``ExtensionArray`` and integer index (:issue:`21257`)
+- :meth:`ExtensionArray.shift` added as part of the basic ``ExtensionArray`` interface. (:issue:`22387`)
```

Comment:
Needs to be :meth:\`pandas.api.extensions.ExtensionArray.shift\`.

Usually we put the period after the issue number.